nestkernel/growth_curve.cpp

```diff
@@ -106,16 +109,17 @@ nest::GrowthCurveGaussian::update( double t,
   // Numerical integration from t_minus to t
   // use standard forward Euler numerics
   const double h = Time::get_resolution().get_ms();
-  const double zeta = ( eta_ - eps_ ) / ( 2.0 * sqrt( log( 2.0 ) ) );
-  const double xi = ( eta_ + eps_ ) / 2.0;
+  const double inv_zeta = ( eta_ - eps_ ) * INV_TWO_SQRT_LOG_TWO;
+  const double xi = ( eta_ + eps_ ) * 0.5;
+  const double inv_tau_Ca = 1.0 / tau_Ca;
```

Comment:
Since `eta_` and `eps_` can only be changed via the `set()` method, I'd suggest to make these three variables data members of the class, create a private method to compute their values and call that method from the constructor and the `set()` method.

And `inv_zeta` needs to be `2 * SQRT_LOG_TWO / ( eta_ - eps_ )` to make the math right (see failing test at https://github.com/nest/nest-simulator/actions/runs/10678447769/job/29595542950?pr=3304#step:23:7813). I'd suggest to keep the factor 2 out of the constant since it is cheap to compute and makes the constant more generally applicable.

Reply:
Completed, thank you. Let's see if the CI tests pass. My benchmarks do verify again now.